### PR TITLE
24 Add layout functionality to any Container based. element

### PR DESCRIPTION
**Pull request by CyberDex**
Implements #24

Nothing changed on the base functionality side.
Just additional option of `.initLayout(options)` to all the Containers.
After the initialisation, you can access it by `container.layout`.

### CODE REVIEW

File: src/Layout.ts [hunk 1]
```diff
@@ -80,13 +80,13 @@ export class Layout extends Container
      * @param options.content - Content of the layout.
      * @param options.globalStyles - Global styles for layout and it's children.
      */
-    constructor(options: LayoutOptions)
+    constructor(options?: LayoutOptions)
```
Comment (CyberDex) [inline src/Layout.ts:83]: Made it optional, to they can be set later

File: src/stories/basic/TransformContainer.stories.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import { argTypes, getDefaultArgs } from '../utils/argTypes';
```
Comment (CyberDex) [inline src/stories/basic/TransformContainer.stories.ts:1]: Story for testing new feature, also the code example for you to check.

File: src/stories/basic/TransformContainer.stories.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            overflow,
+            // text options
+            color,
+            textAlign,
+            fontSize,
+            position,
+            borderRadius
+        });
+
+        this.view.layout.addContent(LOREM_TEXT);
```
Comment (CyberDex) [inline src/stories/basic/TransformContainer.stories.ts:66]: This is it!

**Review by GoodBoyDigital — APPROVED** [review]
this is cool! Gets us where we need to be quickly.
We can refactor layout to not be a container later as this will not affect the API

**Pull request merged**